settings

```diff
@@ -15,11 +15,11 @@
 # limitations under the License.
 
 # Kubernetes branch to get the OpenAPI spec from.
-export KUBERNETES_BRANCH="release-1.12"
+export KUBERNETES_BRANCH="release-1.13"
```

Comment:
14 days past and there's some update on the release-1.13 branch. might be better to re-generate the docs. otherwise LGTM. [https://github.com/kubernetes/kubernetes/pull/73646](https://github.com/kubernetes/kubernetes/pull/73646)

Reply:
done.